apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -205,7 +203,7 @@ protected function setMasterFileShowProperties()
             || $this->showMasterFileMimeType
             || $this->showMasterFileSize
             || $this->showMasterFileCreatedAt
-        ) && null !== $this->masterCopy;
+        ) && isset($this->resource->id);
```

Comment:
@melaniekung sorry, but I re-read the code and if the code makes it this far `isset($this->resource->id)` is always going to be true, so it's unnecessary here.  I think it's better to test `if (!isset($this->resource)` at the beginning of the `execute()` method, and return if `$this->resource` is null.